rbd.c

```diff
@@ -181,32 +181,88 @@ static void tcmu_rbd_image_close(struct tcmu_device *dev)
 	state->image = NULL;
 }
 
+static int update_cmd_time_out(struct tcmu_device *dev,
+					uint32_t timeout)
+{
+	int ret;
+
+	ret = tcmu_set_attribute(dev, "cmd_time_out", timeout);
```

Comment:
Same here as above about handling old kernels. Add support or add a descriptive message so we do not have to handle user questions.